app/routes.php

```diff
@@ -62,6 +62,7 @@
 use Portal\Controllers\FrontEnd\StudentProfilePageController;
 use Portal\Controllers\FrontEnd\TeamPickerPageController;
 use Portal\Controllers\FrontEnd\TrainersPageController;
+use \Portal\Controllers\API\LockApplicantFieldController;
```

Comment:
to keep this consistent, this should be at the bottom of the group of API namespaces, not with the FE ones.